examples/accept/accept.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+const zap = @import("zap");
+
+var gpa = std.heap.GeneralPurposeAllocator(.{
+    .thread_safe = true,
+}){};
+
+fn on_request_verbose(r: zap.Request) void {
+    const allocator = gpa.allocator();
```

Comment:
Wouldn't it make sense to use a FixedBufferAllocator here? Maybe with a 1kB buffer?